packages/destination-actions/src/destinations/google-enhanced-conversions/functions.ts

```diff
@@ -69,3 +83,18 @@ export function convertTimestamp(timestamp: string | undefined): string | undefi
   }
   return timestamp.replace(/T/, ' ').replace(/\..+/, '+00:00')
 }
+
+export function get_api_version(features: Features | undefined, statsContext: StatsContext | undefined): string {
+  const statsClient = statsContext?.statsClient
+  const tags = statsContext?.tags
+
+  if (features && features[FLAGON_NAME]) {
+    tags?.push(`version:${CANARY_API_VERSION}`)
+    statsClient?.incr(`google_api_version`, 1, tags)
+    return CANARY_API_VERSION
+  }
+
+  tags?.push(`version:${API_VERSION}`)
+  statsClient?.incr(`google_api_version`, 1, tags)
+  return API_VERSION
```

Comment:
nit: The following implementation can be a bit more concise.
```suggestion
   const version = features && features[FLAGON_NAME] ? CANARY_API_VERSION : API_VERSION
    tags?.push(`version:${version}`)
    statsClient?.incr(`google_api_version`, 1, tags)
     return version
```